src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2562,10 +2562,9 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                 else
                 {
                     fullPaths.reserve(items.Size());
-                    for (auto item : items)

```

Comment:
ah right, thanks! i think this is an hstring (reference counted, not copied) but it being close to the other one it's better for us to match the style on both sides of the if